apps/sms/style/sms.css

```diff
@@ -1062,3 +1062,19 @@ html[dir="rtl"] .contact-header aside.pack-end {
 .editable-select-mode [contenteditable] {
   -moz-user-select: text;
 }
+
```

Comment:
nit: add a comment to separate this css block; like `/* empty threadlist layout */`